### PR TITLE
Install Prometheus exporters along with the application

### PR DESCRIPTION
**Pull request by gtrkiller**
Delete the exporter container and install them as snaps in the Indico image.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -54,4 +54,4 @@ markers = [
 ]
 
 [tool.pylint]
-disable = "wrong-import-order"
+disable = "wrong-import-order, duplicate-code"
```
Comment (jdkandersson) [inline pyproject.toml:57]: What is the reason for needing this please?

File: src/charm.py [hunk 2]
```diff
@@ -271,7 +262,16 @@ def _config_pebble(self, container: Container) -> None:
         pebble_config = pebble_config_func(container)
         container.add_layer(container.name, pebble_config, combine=True)
         if container.name == "indico":
+            for item in ["statsd", "celery"]:
```
Comment (jdkandersson) [inline src/charm.py:265]: Should this be a tuple?

Comment (jdkandersson) [reply src/charm.py:265]: Also, since there are 2 items, is the `for` loop worth it? It is quite complex compared to the 4 lines of code that are needed?

**Review by merkata — DISMISSED** [review]
Looks good, apart of what David commented on, nothing to add.

File: tests/unit/test_core.py [hunk 3]
```diff
@@ -451,6 +451,7 @@ def test_config_changed_when_saml_target_url_invalid(self, mock_exec):
         self.set_up_all_relations()
         self.harness.set_leader(True)
 
+        # pylint: disable=duplicate-code
```
Comment (jdkandersson) [inline tests/unit/test_core.py:454]: What specifically is pylint complaining about being a duplicate here? In tests it means there is potential for a fixture

Comment (jdkandersson) [reply tests/unit/test_core.py:454]: perhaps worth making it into a fixture now?

**Review by amandahla — DISMISSED** [review]
LGTM :)

Just wanted to see if the images are too much bigger or not than before.

**Review by amandahla — APPROVED** [review]

**Review by jdkandersson — APPROVED** [review]

**Pull request merged**